README.md

```diff
@@ -40,13 +40,13 @@ use std::net::TcpStream;
 use std::str::from_utf8;
 
 // Mock a TCP server listening on port 35642. Note that the mock will only listen on the local interface.
-let tcp_server_mocker = TcpServerMocker::new_with_port(35642).unwrap();
+let server = TcpServerMocker::new_with_port(35642).unwrap();
```

Comment:
Hmmm... Is there any reason for this refactor? 🤔

Reply:
This keeps all examples consistent, and follows the DRY principle - the same line already mentions that it's a TCP, in a short example it is far more readable to consistently use shorter variables. Larger function might benefit from longer var name

Reply:
One more thing I forgot to mention -- this is the same pattern as in [mockito docs](https://docs.rs/mockito/latest/mockito/) - so people already familiar with that popular testing framework will have easier time reading your examples